content/ko/docs/reference/glossary/kubeadm.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Kubeadm
+id: kubeadm
+date: 2018-04-12
+full_link: /docs/admin/kubeadm/
+short_description: >
+  쿠버네티스를 빠르게 설치하고 보안 클러스터를 설정하는 도구
+
+aka: 
+tags:
+- tool
+- operation
+---
+ 쿠버네티스를 빠르게 설치하고 보안 클러스터를 설정하는 도구
+
+<!--more--> 
+
+Kubeadm을 사용하여 컨트롤 플레인과 {{< glossary_tooltip text="워커 노드" term_id="node" >}} 구성 요소를 설치할 수 있다.
```

Comment:
```suggestion
kubeadm을 사용하여 컨트롤 플레인과 {{< glossary_tooltip text="워커 노드" term_id="node" >}} 구성 요소를 설치할 수 있다.
```
No need to assume that the reader is familiar with English capitalization rules. The tool is called “kubeadm” anyway.

Reply:
It was helpful. Thank you.